filer/templates/admin/filer/tools/search_form.html

```diff
@@ -2,33 +2,33 @@
 {% load i18n %}
 <div id="toolbar" class="dataTables_wrapper">
     <div class="row">
-        <form id="changelist-search" action="." method="get" class="col-xs-5 pull-right text-right">
+            <form id="changelist-search" action="." method="get" class="text-right changelist-search-expanded">
             {% csrf_token %}
             <div>
                 {% if is_popup %}
                     <input type="hidden" name="_popup" value="1" />
                     {% if select_folder %}<input type="hidden" name="select_folder" value="1" />{% endif %}
                 {% endif %}
+                {% if current_site %}<input type="hidden" name="current_site" value="{{current_site}}" />{% endif %}
+		<div id="search-container" {% if show_result_count %}style="width:55%"{%endif%}>
```

Comment:
It's better to use classes instead of inline styles. In this case you could do:
````html
{% if show_result_count %}class="with-results-count"{%endif%}
````

````css
.with-results-count{
     width: 55%;
}
````